### PR TITLE
Add 'Logger.list_entries'.

### PR DESCRIPTION
**Pull request by tseaver**
~~Uses #1569 as a base.~~

### CODE REVIEW

**Comment by tseaver** [comment]
@dhermes This one should be a smaller nibble.

File: gcloud/logging/logger.py [hunk 1]
```diff
@@ -128,3 +128,46 @@ def delete(self, client=None):
...
+                           passed, the API will return the first page of
+                           topics.
+
+        :rtype: tuple, (list, str)
+        :returns: list of :class:`gcloud.logging.entry.TextEntry`, plus a
+                  "next page token" string:  if not None, indicates that
+                  more topics can be retrieved with another call (pass that
+                  value as ``page_token``).
+        """
+        log_filter = 'logName:%s' % self.name
```
Comment (dhermes) [inline gcloud/logging/logger.py]: `% (self.name,)`

**Comment by dhermes** [comment]
LGTM other than passing a tuple to format string

**Comment by tseaver** [comment]
I've squashed that in, and will merge after Travis.

**Comment by dhermes** [comment]
Sounds good

**Pull request merged**